README.md

```diff
@@ -1,12 +1,38 @@
-# Git Repository Template
+# Terraform Cloud Development Kit demonstration 
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+The demonstration of [Terraform Cloud Development Kit](https://www.terraform.io/cdktf). 
 
-## Usage
+Contain the implementation on four languages
```

Comment:
```suggestion
Contain the implementation in four languages
```

Reply:
Updated in https://github.com/nimblehq/terraform-cloud-development-kit/pull/10/commits/75f62a9849983988684128ae252bc725471786aa @longnd